### PR TITLE
Added check if user exists

### PR DESCRIPTION
**Pull request by borek2**
@agudek 

Example is within the main.js

If the page is opened with an unknown userName, the user is greeted by a message stating that there is no data available. I also added a spinner to the body during the fetching of the user to complete this check.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ define(['modules/moduleList'], function (dynModules) {
 
         // Set global modules variable to a list of all imported modules after converting pseudo-array to array
         modules = Array.prototype.slice.call(arguments);
+        octopeerService.checkIfUserExists(globalUserName).then(function (exists) {
+            console.log(exists);
```
Comment (lvdoorn) [inline src/main.js]: Should we put a console.log in the new-master?

Comment (agudek) [inline src/main.js]: It was just to show how the user check works. It has been removed in the next commit.

File: src/main.js [hunk 2]
```diff
@@ -1,167 +1,185 @@
...
-                .addClass('col s12 ' + octopeerHelper.getSafeModuleValue(arguments[i], "size"))
-                .appendTo(parentContainer);
-            if (arguments[i].customContainer) {
-                addCustomBody(arguments[i], outerdiv);
-            } else {
-                addDefaultBody(arguments[i], outerdiv);
-            }
+            });
+        } else {
+            console.log(d3.selectAll('.row'));
```
Comment (lvdoorn) [inline src/main.js]: ??? console.log ???

Comment (agudek) [inline src/main.js]: Oops

**Pull request merged**